schema/api.graphql

```diff
@@ -441,7 +441,7 @@ type Query {
     previousTimeslots(time: Int, n: Int, filter: [Int]): [Timeslot] @bind(class: "\\MyRadio\\ServiceAPI\\MyRadio_Timeslot", method: "getPreviousTimeslots")
 
     nowPlaying(
-        "Which source letters (not IDs!) to accept tracklisting from. Defaults to [b, m, o, w, a, s]"
+        "Which source letters (not IDs!) to accept tracklisting from. Defaults to [b, m, o, w, a, s, j]"
```

Comment:
eh? That's just documentation, the change you actually want to make is at https://github.com/UniversityRadioYork/MyRadio/blob/master/src/Classes/ServiceAPI/MyRadio_Track.php#L1635

Reply:
Editing with github online is slow 👍